src/components/Common/RequirementsList/RequirementsList.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames'
+import { useId } from 'react'
+import { Flex } from '../Flex/Flex'
+import styles from './RequirementsList.module.scss'
+import { useComponentContext } from '@/contexts/ComponentAdapter/useComponentContext'
+import SuccessCheck from '@/assets/icons/checkbox.svg?react'
+
+interface RequirementsListProps {
+  requirements: {
+    title: string
+    description: string
+    completed: boolean
+  }[]
+}
+export const RequirementsList = ({ requirements }: RequirementsListProps) => {
+  const Components = useComponentContext()
+  const [id] = useId()
```

Comment:
Nit: i believe you can just do

```suggestion
  const id = useId()
```